src/Service/ValueHandler.php

```diff
@@ -71,144 +84,151 @@ private function getValueByPath(NodeInterface $node, $path, bool $failOnError =
-            foreach ($valuesArray as $v) {
-              $value[] = $v['value'] ? TRUE : FALSE;
+        }
+        break;
+
+      case 'string':
+        $fieldName = $field->getName();
+        switch ($fieldName) {
+          case 'field_partner_organizers':
+            $fieldValues = $field->getValue();
+            foreach ($fieldValues as $v) {
+              $value[] = ['name' => $v['value']];
             }
-          }
-          break;
-
-        case 'string':
-          $fieldName = $field->getName();
-          switch ($fieldName) {
-            case 'field_partner_organizers':
-              $fieldValues = $field->getValue();
-              foreach ($fieldValues as $v) {
-                $value[] = ['name' => $v['value']];
-              }
-              break;
-            case 'field_organiser':
-                $value = ['name' => $field->getValue()[0]['value']];
-              break;
+            break;
 
-            default:
-              $valuesArray = $field->getValue();
+          case 'field_organiser':
+            $value = ['name' => $field->getValue()[0]['value']];
+            break;
 
-              if (1 == count($valuesArray)) {
-                $value = $valuesArray[0]['value'];
-              }
-              elseif (1 < count($valuesArray)) {
-                $value = [];
-                foreach ($valuesArray as $v) {
-                  $value[] = $v['value'];
-                }
+          default:
+            $valuesArray = $field->getValue();
+
+            if (1 == count($valuesArray)) {
+              $value = $valuesArray[0]['value'];
+            }
+            elseif (1 < count($valuesArray)) {
+              $value = [];
+              foreach ($valuesArray as $v) {
+                $value[] = $v['value'];
               }
-              break;
-          }
-          break;
+            }
+            break;
+        }
+        break;
 
-        default:
-          $valuesArray = $field->getValue();
+      default:
+        $valuesArray = $field->getValue();
 
-          if (1 == count($valuesArray)) {
-            $value = $valuesArray[0]['value'];
-          }
-          elseif (1 < count($valuesArray)) {
-            $value = [];
-            foreach ($valuesArray as $v) {
-              $value[] = $v['value'];
-            }
+        if (1 == count($valuesArray)) {
+          $value = $valuesArray[0]['value'];
+        }
+        elseif (1 < count($valuesArray)) {
+          $value = [];
+          foreach ($valuesArray as $v) {
+            $value[] = $v['value'];
           }
-      }
-
+        }
     }
 
     return $value;
   }
 
   /**
    * Make a URL absolute.
+   *
+   * @param string $value
+   *   A uri.
+   *
+   * @return string
+   *   An absolute url.
    */
-  public function makeUrlAbsolute($value) {
-    return file_create_url($value);
+  public function makeUrlAbsolute($value): string {
+    return \Drupal::service('file_url_generator')->generateAbsoluteString($value);
```

Comment:
Use dependency injection to get (and set) the `file_url_generator` service in the constructor.